preproduction-azure-pipelines.yml

```
@@ -44,6 +44,6 @@ jobs:
     displayName: 'npm install'
     inputs:
       verbose: false
-  - script: 'npm install -D @azure/avocado@0.5.0'
+  - script: 'npm install -D @azure/avocado@0.5.1'
```

I think if you switch it to drop the patch version, it will keep using the latest patch (till you just update the package.json version and drop this build)
```suggestion
  - script: 'npm install -D @azure/avocado@0.5'
```

---

Explicit version is better than implicit version here.